openml/setups/functions.py

```diff
@@ -26,9 +26,11 @@ def setup_exists(downloaded_flow, sklearn_model):
     '''
 
     # sadly, this api call relies on a run object
-    openml_param_settings = openml.runs.OpenMLRun._parse_parameters(sklearn_model, downloaded_flow)
-    description = xmltodict.unparse(_to_dict(downloaded_flow.flow_id, openml_param_settings), pretty=True)
-    file_elements = {'description': ('description.arff',description)}
+    openml_param_settings = openml.runs.OpenMLRun._parse_parameters(flow)
```

Comment:
maybe explicitly state in comments that this function raises an error if the flow does not contain all flow ids?